queries.sql

```diff
@@ -49,4 +49,33 @@ GROUP BY species;
 
 SELECT species, AVG(escape_attempts) FROM animals
 WHERE date_of_birth >= '1990-01-01' AND date_of_birth <= '2000-12-31'
-GROUP BY species;
\ No newline at end of file
+GROUP BY species;
+
+SELECT name FROM animals
+JOIN owners ON animals.owner_id = owners.id
+WHERE owners.full_name = 'Melody Pond';
+
+SELECT animals.name FROM animals
+JOIN species ON animals.species_id = species.id
+WHERE species.name = 'Pokemon';
+
+SELECT owners.full_name, animals.name FROM owners
+LEFT JOIN animals ON owners.id = animals.owner_id;
+
+SELECT species.name, COUNT(animals.id) as count FROM species
+LEFT JOIN animals ON species.id = animals.species_id
+GROUP BY species.id;
+
+SELECT animals.name FROM animals
+JOIN species ON animals.species_id = species.id
+JOIN owners ON animals.owner_id = owners.id
+WHERE species.name = 'Digimon' AND owners.full_name = 'Jennifer Orwell';
+
+SELECT animals.name FROM animals
+JOIN owners ON animals.owner_id = owners.id
+WHERE escape_attempts = 0 AND owners.full_name = 'Dean Winchester';
+
+SELECT owners.full_name, COUNT(animals.id) as count FROM owners
+LEFT JOIN animals ON owners.id = animals.owner_id
+GROUP BY owners.id ORDER BY count DESC
+LIMIT 1;
```

Comment:
- [ ] The query gives the wrong result. The correct query should give the name and number of animals of the person that owns the most animals. 👍 

Hint: When creating the `animals` table, that `ID` was not serial, so your current `animals` table has no data for ID. So, I would advise you to either make the `ID` serial or change `count(animals.id)` to `count(animals.species_id)`. 👍 😎